__tests__/app.test.js

```diff
@@ -390,4 +416,43 @@ describe("GET /api/articles/:sort_by?/:order_by?", () => {
       expect(msg).toBe("Bad request")
     })
   })
+})
+
+describe("GET /api/articles/:topic?", () => {
+  test("200: Responds with array of articles filtered by topic", () => {
+    return request(app)
+    .get("/api/articles?topic=mitch")
+    .expect(200)
+    .then(({ body: { articles } }) => {
+      expect(articles.length).toBe(12)
+      articles.forEach((article) => {
+        expect(article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: "mitch",
+          author: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          comment_count: expect.any(String),
+          article_img_url:expect.any(String)
+        })
+      })
+    })
+  })
+  test("200: Responds with an empty array if passed a valid topic with no articles", () => {
```

Comment:
Nice, it'd be better to check it equals [ ] just to make sure it's definitely an array and not an empty string.